client/my-sites/stats/stats-first-view/index.jsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default React.createClass( {
+	render() {
+		return (
+			<div>
+				{ /* TODO: serve this image from proper place */ }
+				<img src="/calypso/images/stats/illustration-stats.svg" />
```

Comment:
With React 15 we can inline the svgs and avoid a request, just render the `<svg>` directly.